### PR TITLE
tcptraceroute: better wording

### PR DESCRIPTION
**Pull request by mt-empty**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Edit: refer to https://github.com/tldr-pages/tldr/pull/3849#discussion_r382273547

### CODE REVIEW

File: pages/linux/tcptraceroute.md [hunk 1]
```diff
@@ -15,11 +15,11 @@
 
 `tcptraceroute {{host}} -p {{source_port}} -s {{source_address}}`
 
-- Set the initial and maximum TTL:
+- Set the first and maximum TTL:
```
Comment (zdroid) [inline pages/linux/tcptraceroute.md]: I would keep initial here.

Comment (mt-empty) [reply pages/linux/tcptraceroute.md]: Refer to https://github.com/tldr-pages/tldr/pull/3849#discussion_r382273547

Comment (zdroid) [reply pages/linux/tcptraceroute.md]: Okay then. My thoughts were that "initial" is more clear in tech documentation.

**Comment by agnivade** [comment]
Thanks. Looks good :+1:

**Pull request merged**